deep_agent/aegra/middleware.py

```
@@ -42,12 +43,32 @@ def validate_api_key(provided_key: str) -> bool:
+def validate_auth_config() -> None:
+    """Validate auth configuration at startup.
+
+    Raises ValueError if AUTH_TYPE is 'jwt' and the JWT secret is
+    missing or too short.
+    """
+    if AUTH_TYPE != "jwt":
+        return
+    if not JWT_SECRET:
+        raise ValueError(
+            "LANGGRAPH_JWT_SECRET must be set when LANGGRAPH_AUTH_TYPE=jwt"
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)